<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="CTLINDT-0792" name="tv.hooq.web.scripts.HOOQ_Web_ActiveUser_Subscription_CreditCard_PH" tests="1" failures="0" timestamp="22 Jan 2018 08:14:44 GMT" time="145.833" errors="1">
  <testcase name="HOOQ_Web_ActiveUser_Subscription_CreditCard" time="145.833" classname="tv.hooq.web.scripts.HOOQ_Web_ActiveUser_Subscription_CreditCard_PH">
    <error type="java.lang.AssertionError" message="Test Case Failed because - Failed to complete HOOQ Web ActiveUser SubscriptionSection">
      <![CDATA[java.lang.AssertionError: Test Case Failed because - Failed to complete HOOQ Web ActiveUser SubscriptionSection
	at org.testng.Assert.fail(Assert.java:94)
	at tv.hooq.web.scripts.HOOQ_Web_ActiveUser_Subscription_CreditCard_PH.TestCaseStatus(HOOQ_Web_ActiveUser_Subscription_CreditCard_PH.java:37)
	at tv.hooq.web.scripts.HOOQ_Web_ActiveUser_Subscription_CreditCard_PH.HOOQ_Web_ActiveUser_Subscription_CreditCard(HOOQ_Web_ActiveUser_Subscription_CreditCard_PH.java:89)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- HOOQ_Web_ActiveUser_Subscription_CreditCard -->
</testsuite> <!-- tv.hooq.web.scripts.HOOQ_Web_ActiveUser_Subscription_CreditCard_PH -->
